python/lsst/meas/deblender/baseline.py

```diff
@@ -82,10 +80,11 @@ def __init__(self, footprint, maskedImages, psfs, psffwhms, log, filters=None,
         None
         """
         # Check if this is collection of footprints in multiple bands or a single footprint
-        try:
-            len(maskedImages)
-        except TypeError:
-            maskedImages = [maskedImages]
+        if not isinstance(mMaskedImage, afwImage.MultibandMaskedImage):
+            mMaskedImage = [mMaskedImage]
```

Comment:
What if it is already a list?

Reply:
It is not allowed to be. The input either needs to be a `MultibandMaskedImage` or a `MaskedImage`, as stated in the docstring.